git_machete/cmd.py

```diff
@@ -569,36 +282,36 @@ def print_annotation(self, b: str) -> None:
             print(self.annotations[b])
 
     def update(self) -> None:
-        cb = current_branch(self.cli_ctxt)
-        if self.cli_ctxt.opt_merge:
+        cb = current_branch(self.git)
```

Comment:
Hmm for a follow-up PR: `current_branch` can probably be a method of `git`

Reply:
Hmm actually I can see quite a few methods in cmd.py that **rely on nothing machete-specific**: at first glance, I'd say everything b/w `currently_rebased_branch_or_none` and `get_latest_checkout_timestamps`, inclusive